website_sale_google_shopping/models/website.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# License, author and contributors information in:
+# __openerp__.py file at the root folder of this module.
+
+from openerp import models, fields
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    google_feed_expiry_time = fields.Integer(
+        string='Feed expiry time',
+        help='Time to keep cached the feed. Set 0 hours to disable cache.',
+        default=24)
+
+    google_image_size = fields.Integer(
+        string='Image size',
+        help='Recomended square images of 800x800 pixels size. The minimum '
+             'will be 100x100 and 250x250 for apparels.',
+        default=800)
+
+    google_use_shipping_settings = fields.Boolean(
+        string='Use shipping settings',
+        help='Uncheck this box if you prefer to use the Google Merchant '
+             'Center shipping settings.',
+        default=True)
+
+    google_shipping_country = fields.Char(
```

Comment:
IMO we should have a `google_shipping_country_id` that relates to `res.country` then this col should `relate=google_shipping_country_id.code`